public/apps/account/password-reset-panel.tsx

```diff
@@ -78,8 +80,7 @@ export function PasswordResetPanel(props: PasswordResetPanelProps) {
 
       await logout(http);
     } catch (e) {
-      setIsNewPasswordInvalid(true);
-      setNewPasswordError(['Invalid new password due to error: ' + e?.body?.message]);
+      setErrorCallOut('Failed to reset password due to error: ' + e?.body?.message);
```

Comment:
A thought: is there a case that e.body.message does not exisit?

Reply:
During my test, I didn't see such cases. Just to be safe.

Reply:
I know `?` is to be safe, my concern is actually if that case happens, user will see `Failed to reset password due to error: undefined` which looks weird and the content of error is not shown anywhere.

Reply:
E.g. failed to connect to server:
![image](https://user-images.githubusercontent.com/63078162/93405209-3c82f700-f841-11ea-9aec-748c5a9725a9.png)

Reply:
I see. What is your suggestion for this? Adding console.error(e)? 

Or we add a empty check for message. If it is empty, we display the existing generic error "You may refresh the page to retry or see browser console for more information."

Reply:
Looks like we already similar logic in tenant switch. I think I can follow the same style.

Reply:
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/dev/public/apps/account/tenant-switch-panel.tsx#L215

Reply:
Yep that looks good.

Reply:
Follow up PR https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/467